setup.sh

```diff
@@ -0,0 +1,5 @@
+git add .
```

Comment:
Your setup.sh should setup the project locally not deploy it on heroku. Installing dependencies is 1 thing that you need to do in setup.sh apart from anything else that has to be done to get the project running on my machine